test/central-config.js

```diff
@@ -87,7 +87,6 @@ test('polling', function (t) {
         t.end()
         res.writeHead(404) // end nicely so we don't get a request-error
         res.end()
-        client.destroy()
         server.close()
```

Comment:
I wasn't able to track down specifically why this line caused an error in your tests -- but it defintely seems related to losing the `destroy` calls I mentioned above.  Let's get those back in there (or talk about their intent so we can come up with a better solution) and see if this call to `destroy` is still necessary.